### PR TITLE
[4.0] Improve updater language strings

### PR DESCRIPTION
**Pull request by bembelimen**
### Summary of Changes
(Hopefully) Improve the updater checkbox texts to be less clumsy. 

### Testing Instructions
Use the nightly XML https://update.joomla.org/core/nightlies/next_patch_list.xml to get an update and see the checkbox texts:

grafik

grafik

### CODE REVIEW

File: administrator/language/en-GB/com_joomlaupdate.ini [hunk 1]
```diff
@@ -125,7 +125,7 @@ COM_JOOMLAUPDATE_VIEW_DEFAULT_NO_DOWNLOAD_URL_DESC="An update to Joomla %1$s was
 COM_JOOMLAUPDATE_VIEW_DEFAULT_NO_LIVE_UPDATE="A new version of the Joomla Update Component is available."
 COM_JOOMLAUPDATE_VIEW_DEFAULT_NO_LIVE_UPDATE_DESC="You must update this component first before you can update Joomla! <a class=\"alert-link\" href=\"index.php?option=com_installer&view=update\">Click here to update the component</a>."
 COM_JOOMLAUPDATE_VIEW_DEFAULT_NON_CORE_PLUGIN_BEING_CHECKED="The system is currently checking these plugins to see if they could cause problems during the update.<br><br>Please be patient while the checks are completed."
-COM_JOOMLAUPDATE_VIEW_DEFAULT_NON_CORE_PLUGIN_CONFIRMATION="I accept the warnings about potentially incompatible extensions and wish to proceed with the update."
+COM_JOOMLAUPDATE_VIEW_DEFAULT_NON_CORE_PLUGIN_CONFIRMATION="Ignore the warnings about potentially incompatible extensions and proceed with the update."
```
Comment (brianteeman) [inline administrator/language/en-GB/com_joomlaupdate.ini]: You are not necessarily ignoring them.

```suggestion
COM_JOOMLAUPDATE_VIEW_DEFAULT_NON_CORE_PLUGIN_CONFIRMATION="Acknowledge the warnings about potentially incompatible extensions and proceed with the update."
```

File: administrator/language/en-GB/com_joomlaupdate.ini [hunk 2]
```diff
@@ -59,7 +59,7 @@ COM_JOOMLAUPDATE_SELF_EMPTYSTATE_TITLE="A new version of the Joomla Update Compo
 COM_JOOMLAUPDATE_SYSTEM_CHECK="System Check"
 COM_JOOMLAUPDATE_TOOLBAR_CHECK="Check for Updates"
 COM_JOOMLAUPDATE_UPDATE_CHECK="Update Check"
-COM_JOOMLAUPDATE_UPDATE_CONFIRM_BACKUP="I've created a backup and my extensions are compatible."
+COM_JOOMLAUPDATE_UPDATE_CONFIRM_BACKUP="I'm prepared for the update and made a backup."
```
Comment (brianteeman) [inline administrator/language/en-GB/com_joomlaupdate.ini]:
```suggestion
COM_JOOMLAUPDATE_UPDATE_CONFIRM_BACKUP="I'm prepared for the update and have made a backup."
```

File: administrator/language/en-GB/com_joomlaupdate.ini (hunk 1 above)
Comment (bembelimen) [reply administrator/language/en-GB/com_joomlaupdate.ini]: Probably we should then also update the confirmation dialog: https://github.com/joomla/joomla-cms/blob/646d9c7b0bfc11b876c719f6c554c6bfcb2a0be8/administrator/language/en-GB/com_joomlaupdate.ini#L128 ?

File: administrator/language/en-GB/com_joomlaupdate.ini (hunk 2 above)
Comment (bembelimen) [reply administrator/language/en-GB/com_joomlaupdate.ini]: Thx

File: administrator/language/en-GB/com_joomlaupdate.ini (hunk 1 above)
Comment (brianteeman) [reply administrator/language/en-GB/com_joomlaupdate.ini]: yes please

I did my first real world update today and this was something I noted to do - you beat me too it.

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 646d9c7b0bfc11b876c719f6c554c6bfcb2a0be8<br /><br />code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35774">issues.joomla.org/tracker/joomla-cms/35774</a>.</sub>

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 646d9c7b0bfc11b876c719f6c554c6bfcb2a0be8<br /><br />Code reviewed. Text changes only which clean up the language strings.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35774">issues.joomla.org/tracker/joomla-cms/35774</a>.</sub>

**Comment by jwaisner** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35774">issues.joomla.org/tracker/joomla-cms/35774</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**